supervisely/nn/inference/inference.py

```diff
@@ -2230,6 +2270,32 @@ def serve(self):
         server = self._app.get_server()
         self._app.set_ready_check_function(self.is_model_deployed)
 
+        if self._is_local_deploy:
+            # Start server
+            server_thread = threading.Thread(target=self._run_server)
```

Comment:
Is the Thread neccessary?

We can start uvicorn only when no predict args are passed